source/clear-linux/guides/tooling/autospec.rst

```diff
@@ -242,7 +242,7 @@ and dependencies in order for autospec to complete a successful build.
 
    .. code-block:: bash
 
-      echo "Intel Corporation" > opae-sdk.license
+      echo "Intel" > opae-sdk.license
```

Comment:
Please change it to "MIT" instead of "Intel" because it's using the MIT license.

Reply:
@bktan8 I have found different information... From https://opae.github.io:

> The OPAE SDK source, unless otherwise noted, is released under a BSD 3-clause license

The [COPYING](https://github.com/OPAE/opae-sdk/blob/master/COPYING) file provides the full details. [BSD-3-Clause](https://spdx.org/licenses/BSD-3-Clause.html) for almost all code. The one exception is the `safe_string` library included in the tree, which is [MIT](https://spdx.org/licenses/MIT.html) licensed.

So, my recommendation is:

```
echo "BSD-3-Clause MIT" > opae-sdk.license
```